types/autobahn/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for AutobahnJS v0.9.7
+// Type definitions for AutobahnJS v0.9.8
```

Comment:
I realize this is your first contribution to Definitely Typed, but there's a couple housekeeping changes you should make as part of this change, as discussed [here](https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md). Could you:
- remove the 'v' prefix and patch version from the version number
- remove the owner websites from the "Definition by" line so that each owner's name is followed by just `<https://github.com/username>`
- remove the line `"dtheader": false` from the `tslint.json` in this folder

It would be great to remove all of the disabled rules from the `tslint.json`, but that's a lot to do for your first contribution. Thanks!

Reply:
I've made the requested changes.

I wouldn't object to removing the disabled rules - just would want a bit more guidance. I'm guessing you'd want me to remove any rules set to `false` (which looks like all of them) and then clean up any linting failures that stem from that?

Reply:
@jrsearles That's correct! Please let me know if you have any questions that come up.